### PR TITLE
Multitude of things

### PR DESCRIPTION
**Pull request by alazymeme**
Moved modules that don't work into an obviously named folder
Slightly changed responses to the giveaway module (and moved it to the broken folder)
Renamed modules to match their module id (to make enabling/disabling easier)

### CODE REVIEW

**Comment by alazymeme** [comment]
I'm looking to move some broken modules to a folder, but it seems like the CircleCI test doesn't passed when I do. Will have to look further into it.

File: pkg/modules/giveaway_start.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ func (c *giveawayCmdStart) Trigger(parts []string, event pkg.MessageEvent) pkg.A
 	}
 
 	if c.m.emoteID == "" {
-		return twitchactions.Mention(event.User, "No emote ID set. Use '!25config emoteid 98374583' to configure this module")
```
Comment (pajlada) [inline pkg/modules/giveaway_start.go]: the "or" part is wrong, the emotename isn't the requirement we check here. it should be a separate `if c.m.emoteName == ""`

Comment (alazymeme) [reply pkg/modules/giveaway_start.go]: Fucking thing deleted.
I've updated it, however I have some doubts. If emoteID is filled, but emoteName is not; would the bot not return "No emote name set. Use '!25config emotename NaM' to configure this module"? Would this not require an `else if` statement of some sort?

Comment (pajlada) [reply pkg/modules/giveaway_start.go]: no, since we use return to break out of the function entirely

**Comment by pajlada** [comment]
Thank you for your contribution!

**Pull request merged**